sdk/spring-sdk/src/it/java/com/example/wiring/SpringSdkWiringIntegrationTest.java

```diff
@@ -109,6 +108,91 @@ public void verifyCounterEventSourceSubscription() {
             new IsEqual(42 + 1));
   }
 
+  @Test
+  public void verifyActionIsNotSubscribedToMultiplyAndRouterIgnores(){
+
+     webClient
+            .post()
+            .uri("counter/counterId2/increase/1")
+            .retrieve()
+            .bodyToMono(Integer.class)
+            .block(timeout);
```

Comment:
```suggestion
    webClient
            .post()
            .uri("counter/counterId2/increase/1")
            .retrieve()
            .bodyToMono(Integer.class)
            .block(timeout);

```

Just a minor formatting thing. This should have be picked by the code formatter. Maybe the Spring SDK module doesn't have it configured?